src/docs/sdk/event-payloads/transaction.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Transaction Payloads
+---
+
+Transactions are used to send tracing events to Sentry.
+
+Transactions must be wrapped in an [Envelope](/sdk/envelopes) and therefore also be sent to the Envelope endpoint.
+
+<Alert title="Note on backwards compatibility" level="info">
+
+Envelopes are a new format and therefore only work on Sentry >= 10. More infromation can be found on the [Envelope](../envelopes) page.
+</Alert>
+
+## Anatomy
+
+A Transaction is basically a [Span](/sdk/event-payloads/span) combined with an [Event](/sdk/event-payloads/). When using 
```

Comment:
Are there fields from `Event` that have no use if set on a `Transaction`?

Reply:
Kinda, so a transaction should not have a stacktrace/exception/threads but we can take care of this in the next iteration.
Not that it will break it's just not defined what happens.